src/codeu/chat/client/ClientUser.java

```diff
@@ -138,8 +150,10 @@ public void updateUsers() {
   }
 
   public static String getUserInfoString(User user) {
-    return (user == null) ? "Null user" :
-        String.format(" User: %s\n   Id: %s\n   created: %s\n", user.name, user.id, user.creation);
+    return (user == null)
+        ? "Null user"
```

Comment:
1) There's quite a bit of places within this project that deal with null values :( I'll definitely keep that in mind, but searching through all of the code and throwing exceptions at every instance of null would be too time consuming for me right now.

2) Done